### ISSUE TITLE
how to set ScaleDenominator when using WebMapTileServiceImageryProvider?

### ISSUE THREAD

**Issue by wkhyh** [CesiumGS/cesium#9133]
Our project use wmts provided by other company. but there only 10 TileMatrix in the TileMatrixSet, and every ScaleDenominator is unusual. where can i set the scale?

**Comment by OmarShehata** [comment]
The ScaleDenominator isn't configurable in CesiumJS. Can you describe your use case a bit more? Are the map tiles you're loading not at the right scale and need to be scaled up or down?

**Comment by wkhyh** [comment]
> The ScaleDenominator isn't configurable in CesiumJS. Can you describe your use case a bit more? Are the map tiles you're loading not at the right scale and need to be scaled up or down?

Yes, it‘s load the wrong tiles, the TileMatrix TileRow TileCol to request are all not right. I think the reason is wmts ScaleDenominator is unusual.

**Comment by wkhyh** [comment]
This is a part of the Capabilities.
![image](https://user-images.githubusercontent.com/35752379/92341323-e9e26600-f0ef-11ea-9a11-2278adf49d2d.png)
